src/main/java/seedu/address/logic/commands/AddReminderCommand.java

```diff
@@ -0,0 +1,94 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_LAST_SEEN;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Reminder;
+
+/**
+ * Adds a reminder to the address book
+ */
+public class AddReminderCommand extends Command {
+
+    public static final String COMMAND_WORD = "remind";
+
```

Comment:
Can add an ALT_COMMAND_WORD here for simplification.

Reply:
added